samples/src/Microsoft.Azure.IIoT.App/src/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
do we need this file?

Reply:
Yes, to install/update the material icons